iwyu_tool_test.py

```diff
@@ -310,6 +310,19 @@ def test_fixup_from_entry_dir(self):
         self.assertEqual('/c057f113f69311e990bf54a05050d914/foobar/Test.cpp',
                          entry['file'])
 
+    def test_unwrap_compile_command(self):
+        """ Wrapping compile commands should be unwrapped. """
+        compilation_db = {
+                'directory': '/home/user/llvm/build',
```

Comment:
Indentation looks a little extreme here, four spaces from `compilation_db` should do.